home_audit.ts

```diff
@@ -84,7 +68,7 @@ function getNestedValidationMessages (homeValues) {
         nested_validate.errors.forEach((error) => {
             const {instancePath, params} = error;
             const errorPath = params.missingProperty ? `${instancePath}/${params.missingProperty}` : instancePath;
-            addErrorMessage(errorMessages[ENUMS.BLOCKER], errorPath, convertAJVError(error))
+            addErrorMessage(errorMessages[ENUMS.ERROR], errorPath, convertAJVError(error))
```

Comment:
this did not catch the zipcode error somehow

Reply:
This commit resolves the issue that ZIP code and a number of other validations weren't showing up in the validation result: https://github.com/hes-pnnl/hes-validation-engine/pull/48/commits/bfea192312edaa02e80da8515eeb406d2c981017